### ISSUE TITLE
Scoring Problems

### ISSUE THREAD

**Issue by Tappyboi** [AnirudhRahul/opsu-Android#22]
Hi uhh ive experience some bugs after updating the app since the new update ruined the scoring system when i try to set  new score to the beatmap it says the score is added but it does not show up in the home screen of opsu also half of my scores were were wiped out in the home screen as well but i check my "replay" files it is still there but still it does not show up in the home screen of opsu which in result i cannot watch some of my replays please fix this. I really like this game and I KNOW it has potential 

 Also there is a glitch with the "experimental slider" where SOMETIMES it shows up as glitchy

**Comment by GODofTheBlackSpace** [comment]
I have this exact same issue.

**Comment by kairusds** [comment]
the experimental slider works fine for me even in 10 star maps, about the scores being deleted you should've clicked the post to leaderboard button to save them or synced it with an account

![Screenshot_2019-11-24-08-54-03-855_fluddokt opsu android](https://user-images.githubusercontent.com/32636532/69487613-02063300-0e98-11ea-8f95-cadcfb8b9971.jpg)

**Comment by Quilvy** [comment]
i think what he wanna say is, about the local high score, i had the same issue as him, the local high score won't show at the homescreen

**Comment by kairusds** [comment]
> i think what he wanna say is, about the local high score, i had the same issue as him, the local high score won't show at the homescreen

homescreen??

**Comment by Quilvy** [comment]
> > i think what he wanna say is, about the local high score, i had the same issue as him, the local high score won't show at the homescreen
> 
> homescreen??

![IMG_20210828_071016](https://user-images.githubusercontent.com/89621011/131199768-01b79a05-bc2b-430b-bb4d-448819768dcc.jpg)
i mean in that local score page, i played over & over but the local score page won't show anything, but the in game score board is work normally smh

**Comment by MFDaus47** [comment]
> > > i think what he wanna say is, about the local high score, i had the same issue as him, the local high score won't show at the homescreen
> > 
> > homescreen??
> 
> ![IMG_20210828_071016](https://user-images.githubusercontent.com/89621011/131199768-01b79a05-bc2b-430b-bb4d-448819768dcc.jpg)
> i mean in that local score page, i played over & over but the local score page won't show anything, but the in game score board is work normally smh
> 

Yeah. I'm also having the same issue. Don't know the cause though. I also noticed that the app doesn't read exactly from the 'replays' folder. This is because when I cleaned my phone's cache. I can't play the latest replay, even though nothing is deleted from the 'replays' folder. Maybe that's where the issue linked?

**Comment by Quilvy** [comment]
agreed
